src/schemas/json/tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "allowTrailingCommas": true,
```

Comment:
Is there a particular reason this was removed? I was so happy that trailing commas were allowed due to this, given that things function perfectly with trailing commas, only to discover that now again they are highlighted as invalid in my IDE.

If this is intentional all fine, just double checking before I remove all trailing commas once more from my code.

Reply:
Oh you're right, I made a mistake removing this. I believe I removed it because the same property didn't exist in `jsconfig.json`, but now i'll add it to both.